cmd/commands/service/command.go

```diff
@@ -120,45 +125,36 @@ func (c *serviceCommand) Run(ctx *cli.Context) (err error) {
 	// 1 for the signal callback
 	// 1 for the node.Wait()
 	// 1 for each of the services
-	c.runErrors = make(chan error, 2+len(serviceTypes))
-	go c.runNode(ctx)
-	c.runServices(ctx, providerID, serviceTypes)
-
-	cmd.RegisterSignalCallback(func() { c.runErrors <- nil })
+	sc.runErrors = make(chan error, 2+len(serviceTypes))
```

Comment:
`errorChannel` and `runErrors` still not aligned and looks duplicated

Reply:
merged it together.